bin/runtime/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph-runtime"
-version = "0.12.0"
```

Comment:
My bad, I forgot this hotfix we do not touch runtime, so ideally we remove this before merge. If this might delay things though, it's fine to leave as it is.